src/components/contact/FormContact.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Button,
+  Group,
+  SimpleGrid,
+  Text,
+  TextInput,
+  Textarea,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { zodResolver } from 'mantine-form-zod-resolver';
+import { z } from 'zod';
+
+export const FormContact = () => {
+  const contactSchema = z.object({
+    email: z.string().email("l'email n'est pas valide"),
+    message: z
+      .string()
+      .min(10, 'le message doit contenir au moins 10 caractères'),
+  });
+
+  const contactForm = useForm({
+    validateInputOnChange: true,
+    initialValues: {
+      name: '',
+      email: '',
+      subject: '',
+      message: '',
+    },
+
+    validate: zodResolver(contactSchema),
+  });
+
+  const isContactButtonDisabled = !contactForm.isValid();
+
+  const handleSubmit = async (event: React.FormEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    throw new Error('Not implemented');
+  };
+
+  return (
+    <form
+      className="contact__form"
+      onSubmit={handleSubmit}
```

Comment:
You have an error of type on this line

![image](https://github.com/FullStackMap/Front/assets/46059868/79d36413-ddc0-47ac-be87-c40ab79290c4)